### PR TITLE
Replace PROPKA pickle with JSON

### PR DESCRIPTION
**Pull request by sobolevnrm**
* Load PROPKA 3.1 data from a pickle rather than JSON to increase platform independence.
* Also fix errors in __init__.py and provide module-level docstring.

### CODE REVIEW

**Comment by sobolevnrm** [comment]
Added fix for typo in tests.

**Comment by sobolevnrm** [comment]
Loosened error tolerance in test.

The PROPKA31 tests fail on Windows 10 Ubuntu 18.04 Windows Subsystem Linux without any changes to the main PROPKA code base.  The relative error when failing is less than 1e-3.  The most recent commit changes the behavior to a test of absolute agreement to a test of relative error less than 1e-3.

File: propka/lib.py [hunk 1]
```diff
@@ -149,68 +236,17 @@ def loadOptions(*args):
     parser = OptionParser(usage)
```
Comment (speleo3) [inline propka/lib.py:236]: `OptionParser` import and instance can be removed

File: Tests/pkacalc_test.py [hunk 2]
```diff
@@ -4,6 +4,11 @@
 from subprocess import call
 import sys
 import unittest
+import logging
+
+
+ACCEPTABLE_ERROR = 0.001
```
Comment (orbeckst) [inline Tests/pkacalc_test.py:10]: I would add the comment describing the rationale for the `ACCEPTABLE_ERROR` (from the comments on th PR).

File: propka/bonds.py [hunk 3]
```diff
@@ -28,11 +29,9 @@ def __init__(self):
         self.max_sq_distance = max(list(self.distances_squared.values())+[self.default_dist_squared])
 
         # protein bonding data
-        self.data_file_name = pkg_resources.resource_filename(__name__, 'protein_bonds.dat')
-
-        data = open(self.data_file_name,'rb')
-        self.protein_bonds = pickle.load(data)
-        data.close()
+        self.data_file_name = pkg_resources.resource_filename(__name__, 'protein_bonds.json')
```
Comment (orbeckst) [inline propka/bonds.py:32]: With this change, increase the minor release number in https://github.com/jensengroup/propka-3.1/blob/91925bb92092d3b180a9c086e81822a3fdec6939/setup.py#L9

File: Tests/pkacalc_test.py (hunk 2 above)
Comment (sobolevnrm) [reply Tests/pkacalc_test.py:12]: Addressed in 9cafb41e060c3b96c145492d95e8e5c60ea0b6fc

File: propka/lib.py (hunk 1 above)
Comment (sobolevnrm) [reply propka/lib.py]: Addressed in 6c67ef152c1f7914f84593c53d9435c1e11790f9.

File: propka/bonds.py (hunk 3 above)
Comment (sobolevnrm) [reply propka/bonds.py:32]: Addressed in 0fbaa6711ff0cd9777363108989cbe71d42dd60e.

**Pull request merged**